components/Icons.es6.js

```diff
@@ -1941,6 +1941,79 @@ function ArrowCircleUp ({ className, width='16', height='16', color = '#766bff',
+    </svg>
+  )
+}
+
+function Microphone ({ className, width='16', height='16', color = '#766bff', viewBox='0 0 16 16' }) {
+  return (
+    <svg width={width} height={height} viewBox={viewBox} fill="none" xmlns="http://www.w3.org/2000/svg" className={className}>
+      <path fillRule="evenodd" clipRule="evenodd" d="M8.16667 2.91667V5.83333C8.16667 7.44415 6.86082 8.75 5.25 8.75C3.63917 8.75 2.33333 7.44415 2.33333 5.83333V2.91667C2.33333 1.30584 3.63917 0 5.25 0C6.86082 0 8.16667 1.30584 8.16667 2.91667Z" fill={color}/>
+      <path fillRule="evenodd" clipRule="evenodd" d="M9.91667 5.25C10.2388 5.25 10.5 5.51117 10.5 5.83333C10.5 8.73285 8.14952 11.0833 5.25 11.0833C2.35051 11.0833 0 8.73285 0 5.83333C0 5.51117 0.261167 5.25 0.583333 5.25C0.905499 5.25 1.16667 5.51117 1.16667 5.83333C1.16667 8.0885 2.99484 9.91667 5.25 9.91667C7.50517 9.91667 9.33333 8.0885 9.33333 5.83333C9.33333 5.51117 9.5945 5.25 9.91667 5.25Z" fill={color}/>
+      <path fillRule="evenodd" clipRule="evenodd" d="M5.25 9.91667C5.57217 9.91667 5.83333 10.1778 5.83333 10.5V12.25C5.83333 12.5722 5.57217 12.8333 5.25 12.8333C4.92783 12.8333 4.66667 12.5722 4.66667 12.25V10.5C4.66667 10.1778 4.92783 9.91667 5.25 9.91667Z" fill={color}/>
+    </svg>
+  )
+}
+
+function More ({ className, width='16', height='16', color = '#766bff', viewBox='0 0 16 16' }) {
+  return (
+    <svg width={width} height={height} viewBox={viewBox} fill="none" xmlns="http://www.w3.org/2000/svg" className={className}>
+      <circle cx="3" cy="8" r="1.5" fill={color} />
+      <circle cx="8" cy="8" r="1.5" fill={color} />
+      <circle cx="13" cy="8" r="1.5" fill={color} />
+    </svg>
+  )
+}
+
+function PencilFilled ({ className, width='16', height='16', color = '#766bff', viewBox='0 0 16 16' }) {
+  return (
+    <svg width={width} height={height} viewBox={viewBox} fill="none" xmlns="http://www.w3.org/2000/svg" className={className}>
+      <path d="M11.4944 0.171376C11.4402 0.117052 11.3758 0.0739523 11.3049 0.0445448C11.2341 0.0151373 11.1581 0 11.0814 0C11.0046 0 10.9287 0.0151373 10.8578 0.0445448C10.7869 0.0739523 10.7225 0.117052 10.6684 0.171376L1.33503 9.50471C1.27062 9.57036 1.22265 9.65032 1.19503 9.73804L0.0283591 13.238C7.2835e-05 13.3254 -0.00718461 13.4182 0.00717742 13.5089C0.0215394 13.5996 0.0571151 13.6857 0.111009 13.76C0.164903 13.8344 0.235593 13.895 0.317325 13.9369C0.399057 13.9787 0.489523 14.0007 0.581359 14.001C0.644069 14.0013 0.70638 13.9911 0.765693 13.9707L4.26569 12.804C4.35342 12.7764 4.43337 12.7285 4.49903 12.664L13.8324 3.33071C13.8867 3.27652 13.9298 3.21215 13.9592 3.14128C13.9886 3.07041 14.0037 2.99444 14.0037 2.91771C14.0037 2.84098 13.9886 2.76501 13.9592 2.69414C13.9298 2.62327 13.8867 2.5589 13.8324 2.50471L11.4944 0.171376Z" fill={color}/>
+    </svg>
+  )
+}
+
+function Nav ({ className, width='16', height='16', color = '#766bff', viewBox='0 0 16 16' }) {
```

Comment:
We have this icon already. It is called Style
<img width="152" alt="image" src="https://github.com/user-attachments/assets/4d47a80f-72d6-45f1-9e8c-a8bb5464ba69" />